charts/ccd-case-management-web/values.preview.template.yaml

```diff
@@ -12,6 +12,7 @@ nodejs:
     CCD_GW_OAUTH2_URL: https://gateway-ccd.aat.platform.hmcts.net/oauth2
     DM_URL: https://gateway-ccd.aat.platform.hmcts.net/documents
     DM_URL_REMOTE: ^https?://(?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|dm-store-aat\.service\.core-compute-aat\.internal(?::\d+)?)/documents
+    ANNOTATION_API_URL: https://ccd-api-gateway-web-pr-243.service.core-compute-preview.internal/em-anno
```

Comment:
hardcoded pr url here?

Reply:
It was for an attempt to test 2 previews together, which didn't work. So, restored the actual value to be there. Thanks.